packages/react/src/component-tests/IcSearchBar/IcSearchBar.cy.tsx

```diff
@@ -11,23 +11,38 @@ import {
   IC_MENU_LI,
   LongOptionLabel,
   Uncontrolled,
+  Basic,
+  Default,
+  Disabled,
+  FullWidth,
+  HelperText,
+  HiddenLabel,
+  Small,
+  OptionsNoFiltering,
+  InAForm,
+  QuerySearchMode,
 } from "./IcSearchBarTestData";
 import {
   BE_VISIBLE,
+  CONTAIN_TEXT,
   HAVE_BEEN_CALLED_WITH,
   HAVE_LENGTH,
+  NOT_EXIST,
 } from "../utils/constants";
 import { setThresholdBasedOnEnv } from "../../../cypress/utils/helpers";
 import "cypress-axe";
 
 const SEARCH_SELECTOR = "ic-search-bar";
+const SEARCH_INPUT = 'input[type="text"]';
 const TEXT_FIELD_SELECTOR = "ic-text-field";
-const DEFAULT_TEST_THRESHOLD = 0.042;
+const DEFAULT_TEST_THRESHOLD = 0.02;
 
-describe("IcSearchBar", () => {
+describe("IcSearchBar end-to-end tests", () => {
```

Comment:
Sorry to be a pedant, but none of our other Cypress tests have 'end-to-end tests' in the description here

Reply:
Some of the newer ones do (IcTextField and IcTabs) and others have something like E2E. Where we're separating out the 'end-to-end' and visual regression tests I think it's good to have the longer description.